lib/src/emitter.dart

```diff
@@ -59,15 +59,24 @@ class DartEmitter extends Object
   /// lint.
   final bool orderDirectives;
 
+  /// If nullable types should be emitted with the nullable suffix ("?").
```

Comment:
Can we expand this a bit? Maybe something like "Null safe syntax should only be enabled if the output will be used with a language version which supports it."

Reply:
Good call, done.